src/chi/components/alert/_toast.scss

```diff
@@ -1,34 +1,33 @@
 @import 'alert-common';
 
+$hr-sizes: (h1, h2, h3, h4, h5, h6);
+
 .chi {
   .m-alert {
     &.-toast {
+      align-items: start;
+      justify-content: space-between;
       max-width: 20rem;
       padding-bottom: 1.1875rem;
```

Comment:
@davidojedacl @jllr same issue here as on -large, the padding-bottom is rendering 19px instead of the same as padding-top. let's change this value to `0.75rem`

Reply:
fixed